tensorlayer/layers/merge.py

```diff
@@ -150,3 +151,60 @@ def __init__(
         # # self.all_drop = list_remove_repeat(self.all_drop)
 
         self.all_layers.append(self.outputs)
+
+
+class ElementwiseLambdaLayer(Layer):
+    """A layer uses a custom function to join multiple layer inputs.
```

Comment:
English Syntax: A layer that use a custom function to combine multiple :class:`Layer` inputs.